contracts/FeeChecker.sol

```diff
@@ -16,30 +16,40 @@ contract FeeChecker is IFeeChecker, Operator {
     //If price < tax_below_price, sending Basis to addresses in feeList will have a fee
     mapping(address => bool) public feeList;
 
-    //Addresses in noFeeList won't pay fees when sending Basis to addresses in feeList
-    mapping(address => bool) public noFeeList; 
+    //Addresses in whiteList are allowed to send transactions to addresses in feeList
+    mapping(address => bool) public whiteList; 
 
     constructor(address _tokenAddress) public {
         tokenAddress = _tokenAddress;
     }
 
     /* ========== VIEW FUNCTIONS ========== */
 
+    //Checks in the Cashv2 contract if the transfer is allowed, blocks transfers to addresses in feeList if sender isn't in whiteList
     function isTransferTaxed(address sender, address recipient) 
         external 
         override
         view
         returns (bool) 
     {
-        if (sender == stakeLockContract || recipient == stakeLockContract) {
```

Comment:
This is to adjust staking reward.  We charge micv2 from the staking reward contract.  pleas remain this logic